packages/runtime-common/realm.ts

```diff
@@ -705,30 +715,30 @@ export class Realm {
         '7d',
         this.#realmSecretSeed,
       );
-      return createResponse(this.url, null, {
+      return createResponse(this.url, this.isPublicReadable, this.isPublicWritable, null, {
```

Comment:
Given that all first three params are properties of realm maybe at this point we can just pass `realm` and read the needed properties in `createResponse`?

Reply:
updated [db68a7e](https://github.com/cardstack/boxel/pull/999/commits/db68a7e8f9d8708b3b5bb72d12e5d8e75c64e7bf)